src/styles/blocks/card.css

```diff
@@ -0,0 +1,95 @@
+.page-card {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  align-items: center;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order